### DISCUSSION TITLE
클로저에 의해 참조된 변수를 뭐라고 하나요?

### DISCUSSION THREAD

**Discussion by publdaze** [publdaze/js-study#90] [클로저]
- 클로저에 의해 참조된 변수를 뭐라고 하나요?

**Comment by publdaze** [comment]
> - 클로저에 의해 참조된 변수를 뭐라고 하나요?

- 자유변수

**Comment by pipisebastian** [comment]
- 자유변수.
- 클로저에 의해 참조되는 상위 스코프의 변수입니다. 즉, 클로저란 자유변수에 묶여있는 함수입니다.
